pkg/controllers/etcd.go

```diff
@@ -59,24 +63,34 @@ func (s *EtcdService) Run(ctx context.Context, ready chan<- struct{}, stopped ch
 	etcdPath := filepath.Join(filepath.Dir(microshiftExecPath), "microshift-etcd")
 	// Not running the etcd binary directly, the proper etcd arguments are handled
 	//		in etcd/cmd/microshift-etcd/run.go.
-	args := []string{"run"}
+	args := []string{}
 
 	// If we're launching MicroShift as a service, we need to do the same
 	//		with etcd, so wrap it in a transient systemd-unit that's tied
 	//		to the MicroShift service lifetime.
 	var exe string
 	if runningAsSvc {
-		args = append([]string{
+		args = append(args,
 			"--uid=root",
 			"--scope",
-			"--property", "BindsTo=microshift.service",
 			"--unit", "microshift-etcd",
+			"--property", "BindsTo=microshift.service",
+		)
+
+		if s.memoryLimit != "" {
+			args = append(args,
+				"--property", fmt.Sprintf("MemoryHigh=%v", s.memoryLimit),
```

Comment:
We're going to need some additional protection for this parameter. I tried `128Mi` and `systemd-run` couldn't understand it.

Perhaps the field should just be an integer for the number of megabytes to allow, instead of the units value?